Naggum.Backend/Matchers.fs

```diff
@@ -0,0 +1,7 @@
+﻿module Naggum.Backend.Matchers
+
+let (|String|Symbol|Object|List|) = function
+    | Reader.Atom (Reader.Object (:? string as s)) -> String s
```

Comment:
Think about it a bit more. Currently Strings are not Objects regarding to that pattern. That's not nice!